workflow-templates/im-build-nuget-package.yml

```diff
@@ -202,7 +202,7 @@ jobs:
         if: env.IS_MERGE_TO_MAIN == 'true'
         uses: im-open/delete-branch-package-versions@v3.1
         with:
-          github-token: ${{ secrets.GH_PACKAGES_TOKEN }} # This is a special org-level secret that needs to be authorized for this repo
+          github-token: ${{ secrets.GITHUB_TOKEN }} # This is a special org-level secret that needs to be authorized for this repo
```

Comment:
The `GH_PACKAGES_TOKEN` is the correct name.  It's a special secret that your repo has to be authorized to use.